code/go/0chain.net/smartcontract/dbs/event/blobber.go

```diff
@@ -267,7 +267,7 @@ func (edb *EventDb) GetBlobbersFromParams(allocation AllocationQuery, limit comm
 	shardSize := sizeInGB(int64(math.Ceil(float64(allocation.AllocationSize) / float64(allocation.NumberOfDataShards))))
 	dbStore = dbStore.Where("read_price between ? and ?", allocation.ReadPriceRange.Min, allocation.ReadPriceRange.Max)
 	dbStore = dbStore.Where("write_price between ? and ?", allocation.WritePriceRange.Min, allocation.WritePriceRange.Max)
-	dbStore = dbStore.Where("max_offer_duration >= ?", allocation.MaxOfferDuration.Nanoseconds())
+	dbStore = dbStore.Where("max_offer_duration >= ?", allocation.MaxOfferDuration.UnixMicro())
```

Comment:
The [blobber.MaxOfferDuration](https://github.com/0chain/0chain/blob/f054bb065050c0d5b761b9ed53f8442042ec42e8/code/go/0chain.net/smartcontract/storagesc/blobber_eventdb.go#L25) was set in nanoseconds when saving to DB, please update it too, otherwise we will get wrong result.